src/openzaak/components/zaken/tests/admin/test_relevantezaakrelatie_model_form.py

```diff
@@ -48,3 +48,25 @@ def test_relevantezaakrelatie_form_clean_throws_exception_if_relevant_zaak_and_r
         form.cleaned_data = {}
         with self.assertRaises(forms.ValidationError):
             form.clean()
+
+    def test_relevantezaakrelatie_form_clean_throws_exception_if_aard_overige_and_overige_relatie_is_not_given(
+        self,
+    ):
+        form = RelevanteZaakRelatieForm()
+        form.cleaned_data = {"_relevant_zaak": 1, "aard_relatie": "overig"}
+        with self.assertRaises(forms.ValidationError):
+            form.clean()
+
+    def test_relevantezaakrelatie_form_clean_does_not_throw_exception_if_aard_overige_and_overige_relatie_are_given(
+        self,
+    ):
+        form = RelevanteZaakRelatieForm()
+        form.cleaned_data = {
+            "_relevant_zaak": 1,
+            "aard_relatie": "overig",
+            "overige_relatie": "overig",
+        }
+        try:
+            form.clean()
+        except forms.ValidationError:
+            self.fail("Exception was raised in clean function when it should not have")
```

Comment:
I think the try except is not strictly necessary, because if the `.clean()` does raise an error, the test will fail due to that error anyway